pkg/apis/serving/v1beta1/predictor_model_test.go

```diff
@@ -36,6 +36,7 @@ func TestGetSupportingRuntimes(t *testing.T) {
 	sklearnRuntime := "sklearn-runtime"
 	pmmlRuntime := "pmml-runtime"
 	mlserverRuntime := "mlserver-runtime"
+	clusterPrefix := "cluster-"
```

Comment:
what's this `clusterPrefix` for ?

Reply:
@yuzisun Previously we receive servingruntimespecs as list, now we receive it as map..It's just to differentiate servingruntimes and clusterservingruntimes names...

Reply:
ok can you help leave a comment or name it to `clusterServingRuntimePrefix`?